Source/EasyNetQ/RabbitAdvancedBus.cs

```diff
@@ -23,7 +23,7 @@ public class RabbitAdvancedBus : IAdvancedBus
         private readonly IConsumerFactory consumerFactory;
         private readonly IEventBus eventBus;
         private readonly IHandlerCollectionFactory handlerCollectionFactory;
-        private readonly ILog logger = LogProvider.For<RabbitAdvancedBus>();
+        private readonly ILogger<RabbitAdvancedBus> logger;
```

Comment:
As I understand it, this is just one use case to demonstrate the approach. In the future, all uses must be replaced.

Reply:
yes sure, this is a POC, if we like this approach, we should carefully implement it, also checking if there are some unit tests around it, etc... and of course covering all the classes that are using logging